os/drivers/lcd/lcd_dev.c

```diff
@@ -68,7 +68,6 @@
 #include <tinyara/fs/ioctl.h>
 #include <tinyara/lcd/lcd_dev.h>
 
-#define MAX_NO_PLANES 3
```

Comment:
This definition is used at some files. Could you let me know where you define this instead of here?

Reply:
We have defined this variable here: include/tinyara/lcd/lcd_dev.h:34:#define MAX_NO_PLANES 3
This variable was being used in two files: "drivers/lcd/lcd_dev.c" and "drivers/lcd/st7701.c"
So, if we add this in include/tinuara/lcd/lcd_dev.h , then both the files will be able to use this.

Reply:
I have defined in lcd_dev.h
path: os/include/tinyara/lcd/lcd_dev.h
I am using this variable in st7701.c driver, and defining again in st7701.c will make it duplicate.